ods_ci/tests/Tests/100__deploy/130__operators/140__serverless_operator/140__serverless_operator.robot

```diff
@@ -42,23 +42,13 @@ Suite Setup
     [Documentation]    Suite Setup
     Set Library Search Order    SeleniumLibrary
     RHOSi Setup
+    Assign Vars According To Product    ${PRODUCT}
```

Comment:
This is not needed (already done in RHOSi Setup)

Reply:
I had to include it as an input param because of:
`Validate Trusted CA Bundles State Managed :: The purpose of this t... | FAIL |
Parent suite setup failed:
Keyword 'RHOSi.Assign Vars According To Product' expected 1 argument, got 0.`

Reply:
I still see this line to be removed